sdk/identity/azure-identity/test/ut/azure_cli_credential_test.cpp

```diff
@@ -320,27 +320,31 @@ TEST(AzureCliCredential, CmdLine)
 TEST(AzureCliCredential, UnsafeChars)
 {
   std::string const Exploit = std::string("\" | echo OWNED | ") + InfiniteCommand + " | echo \"";
+  std::vector<std::string> exploits{Exploit, "space character"};
```

Comment:
Please don't EVER use the phrase "exploits" in test collateral.

Reply:
I agree, 100%. It already existed in the tests, so left it, but will adjust it.

Reply:
I am going to fix this separately, as that's unrelated to this change.

https://github.com/Azure/azure-sdk-for-cpp/issues/5107